### PR TITLE
New Apache Hadoop Yarn Cheat Sheet

### LINKED ISSUE #3813: Apache Hadoop Yarn Cheatsheet:
Apache Hadoop Yarn Cheatsheet
------
IA Page:  http://duck.co/ia/view/apache_hadoop_yarn_cheatsheet 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @Hasil-Sharma

### PR DESCRIPTION
**Pull request by Hasil-Sharma**
## Description of new Instant Answer, or changes
Commands for Apache Hadoop Yarn

## Related Issues and Discussions

Fixes #3813 

---

Instant Answer Page: https://duck.co/ia/view/apache_hadoop_yarn_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/apache-hadoop-yarn.json [hunk 1]
```diff
@@ -0,0 +1,223 @@
+{
+    "id": "apache_hadoop_yarn_cheat_sheet",
+    "name": "Apache Hadoop Yarn",
+    "description": "Commands for Apache YARN",
+    "metadata": {
+        "sourceName": "Apache Hadoop Yarn",
+        "sourceUrl": "https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/YarnCommands.html"
```
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/apache-hadoop-yarn.json]: The link you mentioned here has no information other than your cheat sheet!
Rest LGTM :+1:

Comment (Hasil-Sharma) [inline share/goodie/cheat_sheets/json/apache-hadoop-yarn.json]: Yes, most of it is same but for one of the sections (rmadmin). I left out that one to keep it aesthetically pleasing neither any of the commands in that section are frequently used 😄

Comment (amarlearning) [inline share/goodie/cheat_sheets/json/apache-hadoop-yarn.json]: I think you should mention that link which contains some more information other than that you mentioned in your cheatsheet.

Comment (Hasil-Sharma) [inline share/goodie/cheat_sheets/json/apache-hadoop-yarn.json]: Hmm, I'll search more and add. Other than that ... my IA id is : apache_hadoop_yarn_cheatsheet .... id in my json is : apache_hadoop_yarn_cheat_sheet (and file name is as per that convention) and as far as I know both the id's should be same ... How to name my file properly given this mismatch in id ?

**Comment by Hasil-Sharma** [comment]
Official Documentation is the best repertoire of commands that I could find related to this. 
@amarlearning

**Comment by Hasil-Sharma** [comment]
Any other change I need to make in this ?

**Comment by pjhampton** [comment]
Closing as this PR looks to have gone stale.

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**